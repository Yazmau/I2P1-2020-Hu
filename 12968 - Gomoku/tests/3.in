30
...............
..........b....
......w........
............b..
.www...........
.wbw...........
.wbw...........
..b............
..b............
..b.........w..
..w............
...............
...............
...............
...........bbbb
......b........
..............w
.b.............
...............
...............
w..............
...............
...............
...............
....www........
....wbbbbbw....
....www........
...............
...............
...........bbbb
...............
b..........www.
...........wbw.
...........wbw.
............b..
............b..
.....b......b..
............w..
...............
...............
......w........
...............
...............
......w........
...........bbbb
....bbb...b....
.b..bwwwwwb....
....bbb........
...............
...............
...............
...............
.w.............
...............
..........w....
...............
...............
...............
...............
............www
............www
............wbw
............wbw
.............b.
.....b.......b.
.............b.
........b....w.
...............
w..............
...............
...............
...............
...............
.....w.........
...........bbbb
...............
...............
...............
...............
..............b
...............
...............
...............
.www...........
.wbbbbbw.....b.
.www...........
........w......
.....w.........
...............
...........bbbb
...b..b........
...............
...............
..............w
...............
..........w....
...............
.........bbb...
.........bwb...
.........bwb...
..........w....
..........w....
..........w....
..........b....
............www
...............
...............
......bbb......
......bwwwwwb..
b.....bbb......
...............
...............
.....b.........
...............
...............
...............
...............
..........w....
......w........
............www
...............
...............
...............
........b......
...............
.....w.........
...............
...............
..www..........
..wbw..........
..wbw..........
...b...........
...b.........b.
...b.......w...
...w.......bbbb
...............
...............
...............
.b.............
...............
...............
...............
.......b.....w.
...............
...............
...............
...............
.....w.www.....
.......wbbbbbw.
.......www.bbbb
...............
...............
...............
...b...........
...............
..........b....
...............
......www......
......wbw......
......wbw......
.......b.....w.
.......b.......
....w..b.......
.......w.......
...........bbbb
...............
...............
.........b.....
...............
...............
...............
...............
..b............
...............
...w...........
.bbb...........
.bwwwwwb.......
.bbb...........
...............
.....w......www
....www........
....wbw........
..b.wbw........
.....b.........
.....b.........
....bb.........
.....w.........
...............
...............
...............
......w...w....
...............
...............
...............
...........bbbb
...............
...............
.......www.....
.......wbbbbbw.
.......www.....
...b......b....
...............
...............
...............
...............
...............
.w.............
.....w.........
...............
...........bbbb
...............
......bbb......
......bwb......
..b...bwb......
.......w.......
....b..w.......
.......w.......
.......b.......
.....w.........
...............
...............
......w........
...............
...............
............www
......b........
...............
...........b...
...............
...............
...............
...............
www............
wbbbbbw......w.
www............
....w..........
...............
...............
...............
...........bbbb
...............
...............
.........bwww..
..........wbw..
..........wbw..
...........b...
.......b...b...
.........w.b...
...........w...
...............
...............
...............
...............
....w..........
...........bbbb
...............
...b...........
...............
...............
...............
.bbb...........
.bwwwwwb.......
bbbb...........
.w.............
.....w.........
...............
...............
...............
...............
............www
...............
...............
...............
.www.....b.....
.wbw...........
.wbw...........
..b........b...
..b..........w.
..b............
..w............
...............
...............
...............
........w......
...........bbbb
......b........
...............
...............
...............
...............
...............
.......b.......
...www.........
...wbbbbbw..w..
...www.w.......
...............
...............
...............
...............
...........bbbb
...............
..............b
...............
...............
......b.....w..
...............
...............
...............
....bbb........
....bwb........
....bwb........
.....w.........
...w.w.........
.....w.........
.....b......www
...b...w.......
...............
...............
b..............
...............
...............
...............
...............
...............
...............
.......w.......
...............
..www..........
..wbbbbbw......
..www......bbbb
...............
...............
...............
...............
...............
...............
...............
...............
............bbb
............bwb
.....b.....bbwb
.............w.
.w...........w.
.............ww
...........wwbw
...............
...............
...............
...............
...............
...............
...............
b...w..........
...b...........
...............
.......w.......
...............
......www......
......wbbbbbw..
......www..bbbb
...............
...............
...............
..b............
...............
...............
...............
...............
...b.......www.
...........wbw.
.w.........wbw.
............b..
.....w......b..
............b..
..........bbwbb
...............
...............
...............
...b...........
...............
..........b....
www....w.......
wbbbbbw........
www............
...............
...........w...
...............
...............
...............
...........bbbb
...............
...............
...............
...............
...............
...b...........
..w............
...............
www............
wbwb...........
wbw............
.b.............
.b.............
.b....w........
.w.........bbbb
..........b....
...............
...............
...............
..........b....
...............
...............
...bbb.........
...bwwwwwb.....
...bbb.........
............ww.
...............
...............
...............
............www
...............
..b.....w......
....bbb........
....bwb........
....bwb........
.....w.........
.....w.......b.
.....w.........
.....b.........
...............
.......w.......
...............
...............
...............
............www
...............
...............
...............
...............
...............
...b..www......
......wbbbbbw..
......www......
........w......
...............
...............
......b........
...............
..w............
...........bbbb
