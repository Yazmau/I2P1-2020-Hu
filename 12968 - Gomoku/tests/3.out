black
black
black
white
black
black
white
white
black
black
black
white
black
black
white
black
black
white
black
black
white
black
white
black
black
black
black
white
white
black
